charmcraft/commands/store/__init__.py

```diff
@@ -36,6 +36,10 @@
 
 logger = logging.getLogger('charmcraft.commands.store')
 
+# entity types
+CHARM = 'charm'
```

Comment:
Random suggestion (might need a better name that TYPES)

```python
TYPES = namedtuple("Types", "CHARM BUNDLE")(CHARM="charm", BUNDLE="bundle")
```

Reply:
I considered using an Enum, but it was overkilling (and didn't fit ok). Using a namedtuple here looks like a hack, but fits perfectly with the usage.

I'd drop the uppercases as this is not longer a "const", but more a class, same with attributes.

Improved! Thanks.